src/layout/List/ListComponent.tsx

```diff
@@ -1,133 +1,172 @@
-      (acc, next) => ({ ...acc, [next]: (v) => (typeof v === 'string' ? lang(v) : v) }),
-      {},
-    ),
-    footer: renderPagination(),
-  };
+  function isRowSelected(row: Row): boolean {
+    return JSON.stringify(selectedRow) === JSON.stringify(row);
+  }
 
   return (
     <ComponentStructureWrapper
       node={node}
       label={{ node, renderLabelAs: 'legend' }}
     >
       <div style={{ overflow: 'auto' }}>
-        <LegacyResponsiveTable config={config} />
+        <Table width='100%'>
+          <Table.Head>
+            <Table.Row>
+              <Table.HeaderCell className={classes.headerCell} />
+              {Object.entries(filteredHeaders).map(([key, value]) => (
+                <Table.HeaderCell
+                  key={key}
+                  className={classes.headerCell}
+                  sortable={sortableColumns?.includes(key)}
+                  sort={sortColumn === key ? sortDirection : undefined}
+                  onSortClick={() => {
+                    if (sortColumn === key) {
+                      setSortDirection(sortDirection === 'ascending' ? 'descending' : 'ascending');
+                    } else {
+                      setSortDirection('descending');
+                      setSortColumn(key);
+                    }
+                  }}
+                >
+                  {typeof value === 'string' ? lang(value) : value}
+                </Table.HeaderCell>
+              ))}
+            </Table.Row>
+          </Table.Head>
+          <Table.Body>
+            {filteredRows.map((row) => (
+              <Table.Row
+                key={JSON.stringify(row)}
+                style={{ cursor: 'pointer' }}
+                onClick={() => {
+                  handleRowSelect({ selectedValue: row });
+                }}
+              >
+                <Table.Cell
+                  className={cn({
+                    [classes.selectedRowCell]: isRowSelected(row),
+                  })}
+                >
+                  <RadioButton
+                    aria-label={JSON.stringify(row)}
+                    onChange={() => {
+                      handleRowSelect({ selectedValue: row });
+                    }}
+                    value={JSON.stringify(row)}
+                    checked={isRowSelected(row)}
+                  />
+                </Table.Cell>
+                {Object.entries(row).map(([key, value]) => (
+                  <Table.Cell
+                    key={key}
+                    className={cn({
+                      [classes.selectedRowCell]: isRowSelected(row),
+                    })}
+                  >
+                    {typeof value === 'string' ? lang(value) : value}
+                  </Table.Cell>
+                ))}
+              </Table.Row>
+            ))}
+          </Table.Body>
+        </Table>
+        {pagination && (
+          <div className={cn([classes.pagination, 'fds-table__header__cell'])}>
+            <AltinnPagination
```

Comment:
Maybe we could swap this for the Designsystems `Pagination` component as well if its not too much hassle? 😁 🙏

Reply:
Was a little bit of a hassle as far as I could see, but will make an issue for it! 😄